src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/renderGenericComponent.js

```diff
@@ -135,13 +160,22 @@ module.exports.renderGenericComponent = async function renderGenericComponent()
   store.checkoutConfiguration.session = {
     id: session.id,
     sessionData: session.sessionData,
+    imagePath: session.imagePath,
+    adyenDescriptions: session.adyenDescriptions,
   };
   store.checkout = await AdyenCheckout(store.checkoutConfiguration);
   setCheckoutConfiguration(store.checkout.options);
   setInstallments(store.checkout.options.amount);
   setAmazonPayConfig(store.checkout.paymentMethodsResponse);
   document.querySelector('#paymentMethodsList').innerHTML = '';
 
+  store.checkout.paymentMethodsResponse.paymentMethods.some((pm) => {
+    if (pm.type === 'giftcard') {
+      renderGiftCard(pm);
+    }
+    return null;
```

Comment:
do we need to return anything?

Reply:
linter wanted a return.. "some" function breaks after satisfying the condition once so the return is not doing any harm